app/javascript/greetingSlice.js

```diff
@@ -0,0 +1,41 @@
+// redux/rockets/rocketsSlice.js
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+import axios from 'axios';
+
+const baseUrl = 'api/v1/random_greeting';
+// const apiKey = 'f7b89fe31c77772c4972eea6dc268fcb';
```

Comment:
- [ ] Kindly remove this commented line of code from your codebase. This keeps it clean and professional. You can always revisit old pieces of codes with git history

Reply:
The comment has been removed!!